docs/cms-adapters/CMS_ADAPTERS.md

```diff
@@ -339,3 +339,58 @@ Make sure to expose them in the bundled application by adding them in the `files
 ```
 
 Also make sure to place them in a folder name `migration-scripts` in your packaged app or to set the `migrationScriptFolder` in your `cms.json`.
+
+### Case of dependencies on libraries
+
+If the libraries that you use are providing migration scripts, you need to aggregate them with your own to make the metadata-checks pass.
+
+You will need to specify in your migration script those libraries with their current version.
+
+```json5
+{
+  "$schema": "https://raw.githubusercontent.com/AmadeusITGroup/otter/main/packages/@o3r/extractors/schemas/migration.metadata.schema.json",
+  "version":  "10.0.0",
+  // List of libraries with migration scripts that your project depend on
+  "libraries": {
+    "@mylib/lib": "1.0.0"
+  },
+  "changes": [
+    // The changes specific to your project
+  ]
+}
+```
+
+Then you can automate the aggregation of migration scripts by adding the following builder in your `angular.json` file:
```

Comment:
```suggestion
Then you can automate the aggregation of migration scripts by adding the `@o3r/extractors:aggregate-migration-scripts` builder in your `angular.json` file as follows:
```
Just to ensure that it is clear which builder does the work since you left one migration check builder in your example